source/assets/css/style.css

```
@@ -1,114 +1,118 @@
-	/* Estilização para identificar o espaço do dropdown */
-	background: none;
-	padding: 5px;
-	border-radius: 4px;
-  width: 100%;
-  text-align: center;
-}
-
-.select-wrapper {
-	position: relative;
-	display: inline-block;
+@import url("https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&display=swap");
```

Stylelint (17.4.0)

[error] Expected "url("https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&display=swap")" to be ""https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&display=swap"" (import-notation)

(import-notation)